README.md

```diff
@@ -1 +1,16 @@
-# Hello-microverse
\ No newline at end of file
+Project name
+ -Hello-microverse
+
+ Built with:
+ -html
+ -css
+
+ Getting Started:
+ -Install linters
+ -setup index.html and style.css
+ -write the html code 
+ -write the css code
+
+ bust_in_silhouette Author1:
+ -GitHub:https://github.com/DanUshindi
+ -Twitter:@dan_ushindi
```

Comment:
- [ ] Please, consider using the Microverse template for the README file. The README file is one of the necessary files, which tell visitors of your repo what is about the project and how to work with it - it makes your project more professional. Here is [link](https://github.com/microverseinc/readme-template) for template